src/Compilers/CSharp/Test/Semantic/Semantics/UnsafeTests.cs

```diff
@@ -2485,13 +2485,15 @@ class C
     object f1;
     string f2;
     System.Collections.IEnumerable f3;
-    int? f4;
 }
 ";
             var compilation = CreateCompilation(text);
             var type = compilation.GlobalNamespace.GetMember<NamedTypeSymbol>("C");
 
-            Assert.True(type.GetMembers().OfType<FieldSymbol>().All(field => field.Type.IsManagedType));
+            foreach (var field in type.GetMembers().OfType<FieldSymbol>())
+            {
+                Assert.True(field.Type.IsManagedType, field.ToString());
```

Comment:
This is to improve debuggability of the failing test. You can break on the assert failure and more easily see which field is failing.